packages/editor/editor.css

```diff
@@ -1171,7 +1183,11 @@ input,
 	height: 100%;
 	width: 100%;
 	background-color: var(--color-background);
+	box-shadow:
+		0px 4px 8px -8px inset currentColor,
+		0px 50px 8px -10px inset rgba(0, 0, 0, 0.08);
```

Comment:
This is the "stuck down" shadow, plus a slight extra shadow using the sticky note's color.